src/patient-registration/patient-registration-helper.tsx

```diff
@@ -0,0 +1,43 @@
+export type Patient = {
+  identifiers: [
+    {
+      identifier: string;
+      identifierType: string;
+      location: string;
+    },
+  ];
+  person: {
+    names: [
+      {
+        preferred: boolean;
+        givenName: string;
+        middleName: string;
+        familyName: string;
+      },
+      {
+        preferred: boolean;
+        givenName: string;
+        middleName: string;
+        familyName: string;
+      },
+    ];
+    gender: string;
+    birthdate: Date;
+    birthdateEstimated: boolean;
+    birthtime: Date;
+    addresses: [
```

Comment:
Here would could use the `AddressValue` again. `addresses: Array<AddressValue>;`